package.json

```diff
@@ -208,6 +209,7 @@
 		"concurrently": "5.1.0",
 		"cross-env": "6.0.3",
 		"duplicate-package-checker-webpack-plugin": "3.0.0",
+		"effective-module-tree": "file:./packages/effective-module-tree",
```

Comment:
We put the local file links to `dependencies` rather than `devDependencies`. They are used to wire up the monorepo. Kind of a basic workspaces setup.